pymilo/transporters/tree_transporter.py

```diff
@@ -0,0 +1,110 @@
+# -*- coding: utf-8 -*-
+"""PyMilo SGDOptimizer object transporter."""
+from sklearn.tree._tree import Tree
+
+from .transporter import AbstractTransporter
+from .general_data_structure_transporter import GeneralDataStructureTransporter
+from ..pymilo_param import NUMPY_TYPE_DICT
+
+import numpy as np
+import platform
+
+class TreeTransporter(AbstractTransporter):
+    """Customized PyMilo Transporter developed to handle (pyi,pyx) Tree object."""
+
+    def serialize(self, data, key, model_type):
+        """
+        Serialize instances of the Tree class.
+
+        Record the n_features, n_classes and n_outputs fields of tree object.
```

Comment:
Remove the blank line between the summary descriptions. There should be one blank line between the summary and params sections.

Reply:
![image](https://github.com/openscilab/pymilo/assets/32750909/1fcd5db0-b859-4cfc-8a9d-f7b6e3495c29)

Reply:
the first line is the `Summary` line and the second line is the description line and there should be one blank line between them. we can't have multi-line summary.